### PR TITLE
Introduce Easegress to stable part in kubesphere

### PR DESCRIPTION
**Pull request by suchen-sci**
Hi, this pr aims to introduce [Easegress](https://github.com/megaease/easegress) to Kubesphere. 

We support helm install and uninstall. 

Thanks a lot!

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
Hi Team @kubesphere/sig-microservice,

Could you please help to review this PR?

**Comment by suchen-sci** [comment]
Hi, team, i check the build fail message, but don't really get it... could you please give me some hit so i cam improve? thanks a lot

**Comment by zheng1** [comment]
@xyz-li

File: src/stable/easegress/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: easegress
+type: application
+version: 1.0.0
```
Comment (xyz-li) [inline src/stable/easegress/Chart.yaml:4]: Can you add a maintainer to this chart?

Comment (suchen-sci) [reply src/stable/easegress/Chart.yaml:4]: thanks a lot for your reply! i will update it as soon as possible!

Comment (suchen-sci) [reply src/stable/easegress/Chart.yaml:4]: Hi, sorry for delay... I have add the maintainers for easegress.... please check...

**Pull request merged**